docs/quick_start_guide/using_session_auth.md

```diff
@@ -1,11 +1,13 @@
-# Authentication Flow
+# Using Session Authenticator
+
+**Session** authenticator provides traditional Email/Password authentication.
```

Comment:
**Email/Password** It's probably a term, but my understanding of this at first glance is that Shield only supports email/password (username/password, mobile/password, etc.).

Reply:
Changed to `ID/Password`.